frontend/packages/pipelines-plugin/src/types/repository.ts

```diff
@@ -0,0 +1,25 @@
+import { K8sResourceCommon } from '@console/internal/module/k8s';
+
+export type RepositoryKind = K8sResourceCommon & {
```

Comment:
This is already defined in Abhi's PR, https://github.com/openshift/console/pull/9740
One of you needs to rebase depends on which one gets first